src/units.f90

```diff
@@ -243,6 +243,10 @@ subroutine set_units(ncolumns,numplot,UnitsHaveChanged)
                  units(ix(1:ndim)) = units(icol)
                  unitslabel(ix(1:ndim)) = unitslabel(icol)
                  if (ih.gt.0) then
+                    ibs = index(unitslabel(idivB),'/')
+                    ibc = index(unitslabel(icol),'[')
+                    units(idivB:idivB+3) = units(idivB:idivB+3)*units(ih)/units(icol)
```

Comment:
these lines look dangerous if div B is not present in data - maybe needs an if (idivB > 0) check?

Reply:
I merged and then fixed this